.travis.yml

```diff
@@ -33,17 +33,17 @@ deploy:
     skip_cleanup: true
     on:
       condition: "$NODE_VERSION = 8"
-      repo: thelounge/deb-lounge
+      repo: thelounge/thelounge-deb
```

Comment:
Is this even necessary to specify? I thought travis automatically used the current repo.

Reply:
It's part of the condition. According to travis:

> Deploy only when the build occurs on a particular repository.

I can't remember where I have read that, but this is a safety precaution to avoid forks or else to mistakenly deploy something.

We can leave it or remove it, I just think it doesn't hurt to have this here, but I'm fine with whatever you decide.